lib/config/pacote.js

```diff
@@ -46,9 +46,17 @@ function pacoteOpts (moreOpts) {
     Object.assign(opts, ownerStats)
   }
 
+  if (!!npm.config.get('always-auth')) {
+    if (!opts.auth) { opts.auth = {} }
+    opts.auth.alwaysAuth = true;
+    if (!!npm.config.get('_auth')) {
+      opts.auth._auth=npm.config.get('_auth');
+    }
+  }
+
   npm.config.keys.forEach(function (k) {
     const authMatch = k[0] === '/' && k.match(
-      /(.*):(_authToken|username|_password|password|email|always-auth)$/
+      /(.*):(_authToken|username|_password|password|email|always-auth|_auth)$/
```

Comment:
this regex assume we use 
//registry.npmjs... type of syantax.

could you please change it like so ?

/((.*):)?(_authToken|username|_password|password|email|always-auth|_auth)$/